docker/include/bin/start_faux

```diff
@@ -161,12 +161,18 @@ fi
 if [ -n "${options[ntpv4]}" ]; then
     dhcp_ntp=$(fgrep NTPSERVERS= /run/ntpdate.dhcp)
     ntp_server=`echo $dhcp_ntp | cut -d "'" -f 2`
-    echo Transmitting NTP query to $ntp_server using NTPv4
-    java -jar NTPClient/build/libs/NTPClient-1.0-SNAPSHOT.jar $ntp_server 123 4 2 > ntp.log &
+    (while true; do
+        echo Transmitting NTP query to $ntp_server using NTPv4
+        java -jar NTPClient/build/libs/NTPClient-1.0-SNAPSHOT.jar $ntp_server 123 4 > ntp.log &
+        sleep 8
+    done) &
```

Comment:
this runs a background process inside of a background process.  Previously, it was spawning an infinite number of processes.  What's the rational/reason for this?